File: app/src/main/java/com/quandoo/androidtask/api/Table.java
package com.quandoo.androidtask.api;

import com.google.gson.annotations.SerializedName;

import org.jetbrains.annotations.Nullable;

public class Table {

    @SerializedName("shape")
    String shape;

    @SerializedName("id")
    long id;

    public @Nullable String reservedBy;

    @Override
    public String toString() {
        return "Table{" +
                "shape='" + shape + '\'' +
                ", id=" + id +
                '}';
    }

    public String getShape() {
        return shape;
    }

    public long getId() {
        return id;
    }
}

File: app/src/main/java/com/quandoo/androidtask/data/models/Table.kt
package com.quandoo.androidtask.data.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Table : Parcelable {

    @SerializedName("shape")
    var shape: String? = null
        internal set

    @SerializedName("id")
    var id: Long = 0
        internal set

    var reservedBy: String? = null
    var avatarImageReserve: String? = null

    override fun toString(): String {
        return "Table{" +
                "shape='" + shape + '\''.toString() +
                ", id=" + id +
                '}'.toString()
    }
}
